upload/wagtail_hooks.py

```diff
@@ -16,6 +16,9 @@
 class UploadPackageCreateView(CreateView):
     def form_valid(self, form):
         self.object = form.save_all(self.request.user)
+
+        validate_xml_format(self.object.file.name, self.object.id)
```

Comment:
@rafaelpezzuto esperava que as validações ocorressem somente no background de modo que o usuário só tem acesso ao resultado por meio de visualização de relatório.

Reply:
@robertatakenaka validate_xml_format é um método que fica em tasks.py e que chama uma task.delay (em background). 

Mas talvez o que queremos já é uma capacidade de `observar` pacotes por mudança de estado e, se for o caso, disparar validações (achei que isso fosse a issue #19).

Reply:
@robertatakenaka Concordamos que haverá um método mais genérico `run_validations` que disparará a validação `validate_xml_format`, que, se resultar `True`, disparará, de modo assíncrono várias validações (assets, individual, grouped, etc).

Reply:
sim, no issue 19 você trata isso